src/app/home-page/home-logged-out/home.component.html

```diff
@@ -54,10 +54,14 @@ <h2>An app store for bioinformatics</h2>
             aria-label="Search"
             #search
             type="text"
+            [(ngModel)]="value"
```

Comment:
(keyup.enter) uses search.value. this addition uses value. pick one and use it for both features

Reply:
okay, didn't catch that (keyup.enter) uses search.value, will make the change thank you!